src/pages/Market/components/Item/Item.jsx

```diff
@@ -0,0 +1,28 @@
+import "./Item.css";
+import heart from "../../../../../src/assets/images/heart.svg";
+import fallback from "../../../../../src/assets/images/fallback.png";
+const Item = ({ item, size }) => {
+  const { images, name, price, favoriteCount } = item;
+  return (
+    <div className={`Item ${size}`}>
+      <img
+        src={images && images.length > 0 ? images[0] : fallback}
```

Comment:
💬 여담
images && images.length > 0 대신 images?.length > 0처럼 표현할 수 있습니다. 옵셔널 체이닝을 사용하면 images가 null이나 undefined일 때도 안전하게 length를 확인할 수 있고 가독성 측면에서 더 좋습니다~

```suggestion
        src={images?.length > 0 ? images[0] : fallback}
```